src/users/dto/user.response.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { UserRole } from '@/schema/users.schema';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsMongoId, IsString } from 'class-validator';
+import { Types } from 'mongoose';
+
+export class UserResponseDto {
+  @ApiProperty({ type: String, description: '_id is ObjectID' })
+  @IsMongoId()
+  _id: Types.ObjectId;
+
+  @ApiProperty({ type: String, description: 'username is string' })
+  @IsMongoId()
```

Comment:
???

Reply:
It's should be username (string one) not _id of user in db